addons/web/tests/test_js.py

```diff
@@ -12,7 +12,7 @@ class WebSuite(odoo.tests.HttpCase):
 
     def test_js(self):
         # webclient desktop test suite
-        self.phantom_js('/web/tests?mod=web&failfast', "", "", login='admin', timeout=1800)
+        self.phantom_js('/web/tests?mod=web&failfast&debug=assets', "", "", login='admin', timeout=1800)
```

Comment:
Do you really want to start the tests in debug=assets, there are some difference in behavior.

Reply:
Not necessarily, it's just that debugging tests through headless chrome without this is hell.